android/build.gradle.kts

```
@@ -22,3 +22,31 @@ subprojects {
+            try {
+                val namespaceMethod = android.javaClass.getMethod("getNamespace")
+                if (namespaceMethod.invoke(android) == null) {
+                    val setNamespaceMethod = android.javaClass.getMethod("setNamespace", String::class.java)
+                    var packageName = "com.example.${project.name.replace("-", "_")}"
+                    if (project.name == "flutter_config_plus") {
+                        packageName = "com.chaunguyen.flutter_config_plus"
+                    }
+                    setNamespaceMethod.invoke(android, packageName)
+                    println("Set namespace for ${project.name} to $packageName")
+                }
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)